pkg/bff/db/announcements.go

```diff
@@ -0,0 +1,215 @@
+package db
```

Comment:
This is most of the code in the PR -- database interactions with a new "collection" approach to the key-value store as a document database.